README.md

```diff
@@ -0,0 +1,30 @@
+# LaTeX Project-Manager
+*A project-manager tool to speed up creation process for note-taking*
+---
+## **Description**
+LPM (LaTeX Project Manager) is a simple project manager to generate a new LaTeX project from a template.
+
+## Requirements
+- Python 3.9 at least
+- `art` library (`python3 -m pip install art`)
+
+Tested on *Ubuntu 22.04 LTS*. 
+  
+## Installation
+```
+git clone https://github.com/iScsc/LaTeX-Project-Manager.git
+cd ./LaTeX-Project-Manager/settings
+```
+- Change the "PathToSource" with the *absolute* path to the `source_TeX` folder in the `/settings/config.json` file.
```

Comment:
that is very tideous to do... :thinking: 
maybe you could:
- write `$HOME/path/to/source` inside the file and parse the `$HOME` inside the main code with the `expanduser` function of built-in `os` or `pathlib` libraries
- make your project a real `python` package that you could install with `pip install -e .`. then, the path to the source would not matter :+1: